runtimes/rust/src/lib.rs

```diff
@@ -640,6 +634,12 @@ impl Runtime for RustRuntime {
                 self.remove_started_service(spec);
             }
             InstanceStatus::Migrating(_) => { /* Do nothing. */ }
+            other => {
+                log::warn!(
```

Comment:
I think the runtime should panic here, since this is a sign of the misconfiguration of the node binary.